core/network.rb

```diff
@@ -62,51 +62,67 @@ def loadNodes(config)
 
 
   def self.showKeyFile(name)
-    #TODO refactor with show
+    result_keys = getKeyFile(name)
+    
+    result_keys.each do |hash|
+      $out.out("Keyfile: "+hash["Key"].to_s)
```

Comment:
String "Keyfile: " should be removed because old version of function did not use such format.